web/fido_app/utils.py

```diff
@@ -27,3 +28,32 @@ def validate_email(email):
 def get_display_name(email):
     '''Returns the first part of an email as a display name'''
     return email[: email.index('@')]
+
+
+def get_elapsed_days(startDate, endDate=date.today()):
+    '''
+    Returns the number of days between the provided start and end dates.
+    Defaults to returning the number of days between the start date and today.
+    '''
+    return (endDate - startDate).days
+
+
+def append_to_login_bitfield(bitfield, days_since_last_login):
+    '''
+    Insert (`days_since_last_login` - 1) deactivated bits to the front (most significant) end
+    of the existing `bitfield` integer. Insert one activated bit to the very front.
+    Returns the updated `bitfield` integer.
+    '''
+    if days_since_last_login <= 0:
+        return bitfield
```

Comment:
This function throws an error on my machine because all calls to the function are passing in a date for `days_since_last_login` instead of an integer.

Reply:
Whoops, I forgot to wrap those dates in `get_elapsed_days()`. Should be fixed in [b0fdfdf](https://github.com/team-pass/FIDO-login/pull/70/commits/b0fdfdf2d06ba6067b058da70b9d485f2fd8718b).